### ISSUE TITLE
predict_log_proba is missing on 0.1.7

### ISSUE THREAD

**Issue by Howard-ll** [tensorflow/decision-forests#32]
I heard that predict_log_proba will be supported on 0.1.7 but it is still missing
Please take a look at https://github.com/tensorflow/decision-forests/issues/26

**Comment by achoum** [comment]
The logits are available in v0.1.7, but the signature is different from sklearn:

```python
# Trains a Gradient Boosted Trees that returns logits (assuming the dataset is a binary classification)
model = tfdf.keras.GradientBoostedTreesModel(apply_link_function=False)
```
